long_tests/test_deep_training.py

```diff
@@ -98,18 +99,18 @@ def test_mmdetection():
         first_region = post_model_outputs_response.outputs[0].data.regions[0]
 
         bounding_box = first_region.region_info.bounding_box
-        np.testing.assert_allclose([bounding_box.top_row, bounding_box.left_col, bounding_box.bottom_row, bounding_box.right_col], [0.2178, 0.1606, 0.9614, 0.7952], rtol=0.05, atol=0.05)
+        assert calculate_iou([bounding_box.top_row, bounding_box.left_col, bounding_box.bottom_row, bounding_box.right_col], [0.2289, 0.1643, 1, 0.8023]) > .9, str(post_model_outputs_response)
```

Comment:
Maybe change this function to accept 2 arguments as boxes? Or box and coord

```python
assert calculate_iou(
    bounding_box, 
    Bbox.from_label("expected", 0.2289, 0.1643, 1, 0.8023),
) > .9

Reply:
updated